python/src/tox.ini

```diff
@@ -14,11 +15,21 @@ passenv = TOXENV
 setenv =
     OTEL_PYTHON_TRACER_PROVIDER=sdk_tracer_provider
     OTEL_TRACES_EXPORTER="console_span"
+    ; override CORE_REPO_SHA via env variable when testing other branches/commits than main
+    ; i.e: CORE_REPO_SHA=dde62cebffe519c35875af6d06fae053b3be65ec tox -e <env to test>
+    CORE_REPO_SHA={env:CORE_REPO_SHA:main}
+    CORE_REPO="git+https://github.com/open-telemetry/opentelemetry-python.git@{env:CORE_REPO_SHA}"
 
-install_command = pip install {opts} {packages}
+changedir =
+    test-instrumentation-aws_lambda: {toxinidir}/otel/tests
+
+commands_pre =
+    test: pip install "opentelemetry-test[test] @ {env:CORE_REPO}#egg=opentelemetry-test&subdirectory=tests/util"
+
+    aws_lambda: pip install {toxinidir}/otel[test]
 
 deps = 
-    -r{toxinidir}/otel/tests/test-requirements.txt
+    test: pytest
 
 commands =
-    pytest {toxinidir}/otel/tests/ -s
+    test: pytest {posargs} -s
```

Comment:
@wangzlei Do you know if we need this `-s` (i.e. `--capture=no`) configuration flag for `pytest`?

```
$ pytest --help
usage: pytest [options] [file_or_dir] [file_or_dir] [...]

positional arguments:
  file_or_dir

general:
  --capture=method      per-test capturing method: one of fd|sys|no|tee-sys.
  -s                    shortcut for --capture=no.
```

I don't think we need it so I'd like to remove it because we get an error message even though the tests work fine:

```
test_otel.py sh: -c: option requires an argument
```

Reply:
I added `-s` for printing log in console. If it does not make sense please remove it.